traveler-ios-ui/TravelerKitUI/ViewControllers/BookingOptionsViewController.swift

```diff
@@ -14,6 +14,8 @@ public protocol BookingOptionsViewControllerDelegate: class {
     func bookingOptionsViewController(_ controller: BookingOptionsViewController, didFinishWith purchaseForm: PurchaseForm)
 }
 
+let optionCellIdentifier = "optionCellIdentifier"
```

Comment:
move into class definition, doesn't need to be global

Reply:
I think declaring it globally is a pattern we follow in case it's used somewhere else in the future. Check `stringCellIdentifier` which is also used in its own class but also declared globally.

Reply:
Its not good practice to do that. It will make the code harder to maintain a and follow if you're using identifiers from different files. For the moment, add it to the class scope and in the near future we will refactor our identifiers into a single struct.